compiler/one-cmds/one-quantize

```diff
@@ -233,6 +263,43 @@ def _quantize(args):
         _utils._run(circle_quantizer_cmd, err_prefix="circle_quantizer", logfile=f)
 
 
+def _write_qparam(args):
+    # get file path to log
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    logfile_path = os.path.realpath(args.output_path) + '.log'
```

Comment:
Maybe?
```suggestion
```

Reply:
`dir_path` and `logfile_path` are used below.

Reply:
Ah, sorry.
```python
def _quantize(args):
    if _utils._is_valid_attr(args, 'force_quantparam'):
        # write quantization parameters
        _write_qparam(args)
        return

    # get file path to log
    dir_path = os.path.dirname(os.path.realpath(__file__))
    logfile_path = os.path.realpath(args.output_path) + '.log'
```
If condition can be hoisted.